mysql-test/suite/galera/r/galera_last_committed_id.result

```diff
@@ -1,38 +1,35 @@
 connection node_2;
 connection node_1;
-SELECT WSREP_LAST_WRITTEN_GTID() = '00000000-0000-0000-0000-000000000000:-1';
-WSREP_LAST_WRITTEN_GTID() = '00000000-0000-0000-0000-000000000000:-1'
-1
-wsrep_last_committed_id_match
-1
-connect node_1a, 127.0.0.1, root, , test, $NODE_MYPORT_1;
-connection node_1a;
+SELECT WSREP_LAST_WRITTEN_GTID();
```

Comment:
What is the purpose of a `WSREP_LAST_WRITTEN_GTID()` function? Isn't this the same as [@last_gtid](https://mariadb.com/kb/en/library/gtid/#last_gtid)?

Reply:
WSREP_LAST_WRITTEN_GTID will output last GTID executed & committed for connection to database. In general, for all following comments related to WSREP_GTID functions - they work even binlog is disabled, i assume that original authors found this useful to implement. This patch tries to move away from galera GTID representation (uuid:seqno) and switch to MariaDB GTID format (domain-server-seqno). In case when binlog is enabled, you could match WSREP_GTID with GTID found in binlog files.

Reply:
In the original case where Galera GTIDs where different there would of been an incompatibility (which is my assumption of original authors intent), now it unfortunately looks like putting in a Galera branding on a function where a holistic integrated solution is what users want.

I can't see a useful case for Galera GTID function exposure to be different from existing MariaDB GTID exposure.

Implementing an additional Galera specific function in frameworks from Drupal to Larvel, to SQLAlchemy, to Django, and with bespoke enterprise applications. It should be possible to write applications and frameworks to handle read delay without regard to the replication technology.

Appreciate moving away from Galera GTID implementation to MariaDB GTID putting the "G" back in GTID.